src/test/java/ut/com/isroot/stash/plugin/ConfigValidatorTest.java

```diff
@@ -29,6 +29,7 @@
     @Before
     public void setup() {
         MockitoAnnotations.initMocks(this);
+        System.setProperty("line.separator", "\n");
```

Comment:
Hey @wadahiro , just jumping back into PRs :-) Would it make sense (or is it possible) to set this in `pom.xml` for all tests? Maybe part of `maven-surefire-plugin` configuration? If we set that there, then it would also fix future tests that might have the same problem when running in Windows.